maquetado/styles.scss

```diff
@@ -1,3 +1,5 @@
 body {
   font-family: 'Roboto', sans-serif;
+  margin: 10px;
```

Comment:
Delete this you should do this with `padding` property. Check this [article](https://medium.com/wolox-driving-innovation/what-margin-padding-taught-me-after-50-projects-6878b53c903f)